aiida/engine/processes/calcjobs/calcjob.py

```diff
@@ -841,6 +841,7 @@ def presubmit(self, folder: Folder) -> CalcInfo:
 
         # I create the job template to pass to the scheduler
         job_tmpl = JobTemplate()
+        job_tmpl.shell_type = computer.get_transport_class().shell_type
```

Comment:
Francisco had a comment regarding the shell type. @ramirezfranciscof correct me if I'm wrong, but you stated that aiida-core expects the host to work in bash. Windows machines will be running Powershell, so we should allow the `verdi computer` command to take a shell argument, i.e. `shell_type`. Is this correct?

Reply:
I think that conceptually, the shell type is a property of the _computer_ not of the transport which is the manner in which we communicate with the computer. So I think this would require adding a `shell_type` attribute to the `Computer` ORM class which can be configured when a computer is setup. When a transport is opened, the shell type needs to be defined, which when opening through a `Computer` the computer passes its `shell_type`.

Reply:
yes I agree